easybuild/easyblocks/b/binutils.py

```diff
@@ -68,7 +71,10 @@ def configure_step(self):
             # https://github.com/easybuilders/easybuild-easyconfigs/issues/10056
             # Escaping: Double $$ for Make, \$ for shell to get literal $ORIGIN in the file
             libdirs = [r'\$\$ORIGIN/../lib']
-            for libdir in ['/usr/lib', '/usr/lib64', '/usr/lib/x86_64-linux-gnu/']:
+            for libdir in ['lib', 'lib64', os.path.join('lib', 'x86_64-linux-gnu')]:
+
+                libdir = os.path.join(sysroot, 'usr', libdir)
+
                 # also consider /lib, /lib64
                 alt_libdir = libdir.replace('usr/', '')
```

Comment:
Should probably explicitly use
```
alt_libdir = os.path.join(sysroot, libdir)
```
since you went to the bother of removing the slash everywhere else